python/lib/cloudutils/serviceConfig.py

```diff
@@ -23,6 +23,29 @@
 import os
 import shutil
 
+# exit() error constants
+Unknown = 0
+CentOS6 = 1
+CentOS7 = 2
+Ubuntu = 3
+RHEL6 = 4
+RHEL7 = 5
+
+#=================== DISTRIBUTION DETECTION =================
+if os.path.exists("/etc/redhat-release"):
+    version = file("/etc/redhat-release").readline()
+    if version.find("Red Hat Enterprise Linux Server release 6") != -1:
+        distro = RHEL6
+    elif version.find("Red Hat Enterprise Linux Server 7") != -1:
+        distro = RHEL7
+    elif version.find("CentOS release 6") != -1:
+        distro = CentOS6
+    elif version.find("CentOS Linux release 7") != -1:
+        distro = CentOS7
+elif os.path.exists("/etc/legal") and "Ubuntu" in file("/etc/legal").read(-1): distro = Ubuntu
```

Comment:
instead of `/etc/legal` perhaps `/etc/lsb-release` could be used for Ubuntu/Debian.

Reply:
Hi @rhtyd 
yes we can change it... i can change it oaky?